sass/DynamicHierarchicalFacet/_DynamicHierarchicalFacetValues.scss

```diff
@@ -87,16 +87,17 @@ button.coveo-dynamic-hierarchical-facet-all {
 
 .coveo-dynamic-hierarchical-facet-value-arrow {
   position: absolute;
+  top: 6px;
   left: 0;
 }
 
-$arrow-svg-dimensions: 25px;
+$arrow-svg-dimensions: 15px;
 .coveo-dynamic-hierarchical-facet-value-arrow-svg {
+  transform: rotate(90deg);
```

Comment:
Could you add a nice animation on `transform` ?

 Or is the arrow removed/added back to the DOM each time we redraw, so the animation would not work ?

Reply:
I'm only reusing the arrow down and rotating it so it points left.